pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[build-system]
+requires = ["setuptools>=40.8.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "basyx-python-sdk"
+version = "0.1.0"  # Replace with your actual version
+description = "The Eclipse BaSyx Python SDK, an implementation of the Asset Administration Shell for Industry 4.0 systems"
+authors = [
+    {name = "The Eclipse BaSyx Authors", email = "example@example.com"}  # Replace with a real email
```

Comment:
The E-Mail needs replacing, I wonder what we should put there.

Reply:
Tnx. Which email should I write here?